comments/classified.csv

```diff
@@ -2511,6 +2511,212 @@ Yo what the fuck is your problem fucking around with my page! leave it the fuck
+My opinion is just make the change to satisfy the linter. While I see the argument for the 'define as close to usage as possible' it's not hard to use GoToDef occasionally vs adding gross comments,0,0.5
+"Your code is also missing one important point: you're only propagating the new direct references. If A -> B and C -> D, and I'm adding B -> C, then you need to get D added too. My first loop computes you need to include D, also accounting for the fact we might not know C's transitive references and thus instead need to propagate ""we don't know""",0,0.5
+Ack.  That feeling when you realize you're an idiot :D,1,0.5
+item3 is a pretty terrible name.  it also seems incorrect given that Item3 looks to be a DeclarationExpressionSyntax.  Name tuple properties are a good thing :),0,0.5
+"Yeah, I need to figure out better wrapping. That said, they all look terrible",0,0.5
+"The above one, the lambda-ness of it is getting insane. There are so many ({{({()})}}}) that I think it will lead to mistakes and issues more than separating out the method. You should prescribe a separate method if possible at this point",0,0.5
+Base time running #code,0,0.5
+Time running after adding this new test,0,0.5
+"@github I would consider keeping this test as #code considering that the test above, #code, which uses a similar code minus the insane amount of dictionary instances, is",0,0.5
+already signaled to run on #code,0,0.5
+"holy crap, this is what does the LRU magic",0,0.5
+* [ ] How are we lanning on dealing with the numerous TabView quirks in v2? Just ship it as-is and fix in post? Or manually adjust the styling now,0,0.5
+* [ ] EX the close button hover is weirdly wide,0,0.5
+* [ ] the tab text is bold,0,0.5
+"* [x] Icons throughout the app are still on the Segoe MDL2, not Segoe Fluent",0,0.5
+* [x] Tab color picker looks fine,0,0.5
+* [x] Tab renamer looks fine (if maybe needs a little padding...),0,0.5
+* [x] Window Renamer / ID toast looks fine,0,0.5
+"* [x] About looks... Fine? The links aren't underlined anymore, but that might just be how WinUI works these days",0,0.5
+* [x] #codes look totally ~~insane~~ transparent,0,0.5
+. image](ht,0,0.5
+"I'm stupid, that should have been and #endif but is better with the NoXYZ attrs",0,0.5
+"Please excuse me if this is a stupid question, but why can't you just call the #code method here",0,0.5
+"Build what? Oh crap, I had built it from VS and it worked. I will take a look soon. 😥",0,0.5
+"That's fair, it could be made internal but it feels a little gross since users can new it up and use it in their own factory. Though that's gross as well",0,0.5
+"Crap - you're right... I didn't even think about that... yeah - that won't work. I'll try, but I'm pretty sure you're right",0,0.5
+gross,0,0.5
```

Comment:
I think this one should be negative:
```suggestion
gross,1,0.5
```

Reply:
looks like these comments are already been classified, this is at line 2216/2215

Reply:
So i updated the classified again and removed all redundant comments